assets/src/edit-story/components/inspector/prepublish/checklistTab.js

```diff
@@ -186,13 +201,18 @@ const ChecklistTab = (props) => {
     <>
       {showHighPriorityItems && (
         <SimplePanel
-          collapsedByDefault={false}
+          collapsedByDefault={isHighPriorityDisabled}
+          isDisabledToggle={isHighPriorityDisabled}
```

Comment:
Nit: I think it makes more sense to me to name this `isToggleDisabled`. Maybe even `disabled` works here too, but it's not a big deal.

Reply:
First one would be consistent with `isRecommendedDisabled`

Reply:
guess i had yoda on the mind while writing this. i'll fix it.